Modules/Security/CorsPolicy.cs

```diff
@@ -10,7 +10,15 @@ public static class CorsPolicy
         /// Creates a policy that does not restrict browsers from interacting
         /// with the requested resources.
         /// </summary>
-        public static CorsPolicyBuilder Permissive() => new CorsPolicyBuilder().Default(new OriginPolicy(null, null, null, true, 86400));
+        /// <param name="defaultAuthorizationHeader">Indicate if the header Authorization should be in 'Access-Control-Allow-Headers'. Default value=false</param>

```

Comment:
"Default value=false", but default value is set to `true`?

Reply:
Regarding the naming, probably something like `allowAuthorization` would express the intention more clearly?

Reply:
It was a comment mistake. Fixed !!. The variable has been changed => allowAuthorization